### PR TITLE
Benjamin/styling floor plan component

### PR DESCRIPTION
**Pull request by Benjamin-GitHub**

### CODE REVIEW

File: client/src/pages/FloorPlan.js [hunk 1]
```diff
@@ -3,25 +3,34 @@ import "./FloorPlan.css";
...
+	}`;
 	return (
 		<>
-			<g transform={`rotate(${desk.r} 75 75)`}>
+			<g className={deskColor}>
 				<rect
-					className={deskColor}
-					width={150}
-					height={150}
+					// className={deskColor}
```
Comment (gregdyke) [inline client/src/pages/FloorPlan.js:13]: don't commit commented code

File: client/src/pages/FloorPlan.js [hunk 2]
```diff
@@ -3,25 +3,34 @@ import "./FloorPlan.css";
 
 const Desk = ({ desk, booking }) => {
 	const deskColor = `${booking ? "booked-desk" : "free-desk"}`;
+	const placeMarkerColor = `${
```
Comment (gregdyke) [inline client/src/pages/FloorPlan.js:6]: placeMarkerClass would be a better name

File: client/src/pages/FloorPlan.js [hunk 3]
```diff
@@ -3,25 +3,34 @@ import "./FloorPlan.css";
 
 const Desk = ({ desk, booking }) => {
 	const deskColor = `${booking ? "booked-desk" : "free-desk"}`;
```
Comment (gregdyke) [inline client/src/pages/FloorPlan.js:5]: this is the same as deskColor = booking ? "booked-desk" : "free-desk"; 

also deskClass, not color?

**Review by gregdyke — APPROVED** [review]
Looks amazing!

Let's approve this for now and address comments in a 2nd PR:
- variable naming could be improved
- the cursor changes to a text cursor instead of a pointer over the name/number -> can make it a pointer throughout?
- the hover doesn't work when hovering over the text cursor. could you fix that too?

**Comment by Benjamin-GitHub** [comment]
Thanks @gregdyke I'll do all this changes at my next PR.

**Pull request merged**